dependent/european-journal-of-epidemiology.csl

```diff
@@ -11,7 +11,7 @@
     <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <issn>0393-2990</issn>
+    <issn>1573-7284</issn>
     <eissn>1573-7284</eissn>
```

Comment:
Okay, that is occuring several times and can be easily solved:
 * european-journal-of-epidemiology --> wrong replacement for issn (the old issn 0393-2990 <a href="http://www.springer.com/public+health/epidemiology/journal/10654">seems correct</a>)
 * aging-clinical-and-experimental-research --> delete issn (only eissn)
 * history-and-philosophy-of-the-life-sciences --> issn should be 0391-9714
 * springer-science-reviews --> delete issn (only eissn)